state_machine/applications/flight/lib/mathutils.py

```diff
@@ -38,43 +38,31 @@ def hat(v):
     """Converts v to a matrix such that hat(v)w = cross(v, w)"""
     if not isinstance(v, ndarray):
         v = array(v)
-    if v.shape == (3, 1):
-        return array([
-            [0,         -v[2][0],   v[1][0]],
-            [v[2][0],    0,        -v[0][0]],
-            [-v[1][0],   v[0][0],   0]])
-    elif v.shape == (3,):
-        return array([
-            [0,      -v[2],   v[1]],
-            [v[2],    0,     -v[0]],
-            [-v[1],   v[0],   0]])
-    else:
-        raise ValueError("v must be a 3x1 numpy array")
+    return array([
+        [0,      -v[2],   v[1]],
```

Comment:
I'm glad you made this change. In my experience it's the more standard way of doing things in numpy - vectors are 1d so there is no notion of "column" or "row".

Reply:
Yeah, I didn't realize it was this way in numpy until much later.